featuretools/feature_base/feature_base.py

```diff
@@ -327,9 +334,8 @@ class DirectFeature(FeatureBase):
 
     def __init__(self, base_feature, child_entity):
         base_feature = _check_feature(base_feature)
-        if base_feature.expanding:
-            self.expanding = True
         self.base_feature = base_feature
+        self.primitive = base_feature.primitive
```

Comment:
hmm, not sure about this. i would this our code shouldn't ever try access the primitive property of a direct feature